packages/extensions/modelerfour/package.json

```diff
@@ -11,14 +11,14 @@
   "main": "dist/exports.js",
```

Comment:
Does this need to point to `dist/src/main.js` now?  I don't think `dist/exports.js` exists